app/build.gradle

```diff
@@ -49,6 +49,6 @@ dependencies {
     testImplementation "io.mockk:mockk:1.10.0"
     implementation 'com.squareup.retrofit2:retrofit:2.9.0'         //added internet permissions to androidmanifest.xml
     implementation 'com.squareup.retrofit2:converter-gson:2.9.0'//implementations
-    implementation 'com.google.firebase:firebase-analytics:17.2.2'
+    implementation 'com.google.firebase:firebase-analytics:17.4.4'
```

Comment:
Upgraded Firebase Analytics dependency per the suggestions here: https://firebase.google.com/support/release-notes/android#kotlin+ktx